components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/main/java/org/wso2/carbon/identity/api/server/idp/v1/core/ServerIdpManagementService.java

```diff
@@ -951,6 +954,10 @@ private IdentityProviderTemplateListResponse createIDPTemplateListResponse(
                     idpTemplateListItem.setDisplayOrder(
                             Integer.valueOf(idpTemplate.getPropertiesMap().get(PROP_DISPLAY_ORDER)));
                 }
+                if (idpTemplate.getPropertiesMap().containsKey(PROP_SERVICES)) {
+                    idpTemplateListItem.setServices(Arrays.asList(
+                            idpTemplate.getPropertiesMap().get(PROP_SERVICES).split(",")));
```

Comment:
check whether idpTemplate.getPropertiesMap().get(PROP_SERVICES) is not null before splitting